docs/production-deployment/cloud/nexus/getting-started.mdx

```diff
@@ -68,3 +70,13 @@ Please note the following points:
   The new Namespace may not have the underlying Workflow.
 
 In these scenarios, Temporal can provide guidance on different migration strategies.
+
+<RelatedReadContainer>
```

Comment:
We generally have kept the Related Read Containers only for the Evaluate section. This could be turned into a [tip admonition](https://docusaurus.io/docs/markdown-features/admonitions).

Reply:
I searched on `ReleatedReadContainer` and saw a lot of usage in the encyclopedia, so was following those patterns. Is there a good example of a tip admonition that shows a list of related items?

Reply:
I'm going to start actively removing those In the Encyclopedia. For the Encyclopedia, what's the actual next step you want people to take after reading this doc? If you don't think people are in the right place, what link do you want to put to get them there?

Reply:
Sounds good, will update.

Reply:
fixed